app/src/main/java/fr/free/nrw/commons/category/CategorizationFragment.java

```diff
@@ -194,6 +195,20 @@ public void onActivityCreated(Bundle savedInstanceState) {
         onCategoriesSaveHandler = (OnCategoriesSaveHandler) getActivity();
         getActivity().setTitle(R.string.categories_activity_title);
     }
+    void removeduplicates(RVRendererAdapter<CategoryItem> categoriesAdapter){
```

Comment:
`private` I guess? Please also mind the line spacing.

Reply:
Default class visibility is `private` so I am okay with the method signature not being explicit about it. I agree with the line spacing comment though.